index.d.ts

```diff
@@ -50,10 +50,10 @@ declare module 'react-intersection-observer' {
     triggerOnce?: boolean
 
     /** Call this function whenever the in view state changes */
-    onChange?(inView: boolean | React.FormEvent<HTMLElement>): void
+    onChange?: (inView: boolean) => void
```

Comment:
Ditto for the syntax.
Also, I could not find any reference to an event being passed to this function. Both the Flow types and the source code show nothing of this type.

Reply:
See #39 for the FormEvent. Not sure if it's still an issue?